bokeh/util/callback_manager.py

```diff
@@ -62,10 +74,13 @@ class EventCallbackManager:
 
     '''
 
+    document: Document | None
+    id: ID
+    subscribed_events: List[str]
     _event_callbacks: Dict[str, List[EventCallback]]
 
-    def __init__(self, *args, **kw) -> None:
-        super().__init__(*args, **kw)
+    def __init__(self, *args: Any, **kw: Any) -> None:
+        super().__init__(*args, **kw)  # type: ignore
```

Comment:
also not sure if there is a better way to handle mixins. I guess people have settled on having the mixin also inherit from the base, and also carefully controlling MROs, and that just seems like a mess.